extractor/test/test_extractor.py

```diff
@@ -0,0 +1,121 @@
+from pytest import raises
```

Comment:
these tests are duplicated  extractor/test/extract/test_extractor.py and  extractor/test/test_extractor.py or am I missing something ?

Reply:
Oops